plugins/providers/helm/release.go

```diff
@@ -18,45 +18,46 @@ import (
 var ErrReleaseNotFound = errors.New("release not found")
 var ErrChartNotApplication = errors.New("helm chart is not an application chart")
 
-type releaseConfig struct {
+type ReleaseConfig struct {
 	// Name - Release Name
-	Name string `valid:"required"`
+	Name string `json:"name" mapstructure:"name"`
 	// Repository - Repository where to locate the requested chart. If is a URL the chart is installed without installing the repository.
-	Repository string `valid:"required"`
+	Repository string `json:"repository" mapstructure:"repository" default:"https://odpf.github.io/charts/"`
 	// Chart - Chart name to be installed. A path may be used.
-	Chart string `valid:"required"`
+	Chart string `json:"chart" mapstructure:"chart" default:"firehose"`
```

Comment:
This provider is supposed to be used by many modules, we need a way to set defaults required for `firehose` in the firehose module itself.